_header

```diff
@@ -0,0 +1,10 @@
+Cache-Control: no-cache, no-store, max-age=0, must-revalidate
+Pragma: no-cache
+Expires: 0
+X-Content-Type-Options: nosniff
+X-Frame-Options: DENY
+X-XSS-Protection: 1; mode=block
+referrer-policy: no-referrer, strict-origin-when-cross-origin
+Strict-Transport-Security: max-age=31536000 ; includeSubDomains
+content-security-policy: default-src 'none'; script-src 'self' 'unsafe-inline' ajax.googleapis.com cdn.syndication.twimg.com; connect-src 'self'; img-src 'self' data: pbs.twimg.com *.travis-ci.org travis-ci.org; style-src 'self' 'unsafe-inline' fonts.googleapis.com; font-src 'self' data: fonts.gstatic.com; object-src 'none'; frame-ancestors 'none';report-uri https://unic.report-uri.com/r/d/csp/enforce; report-to csp
+Report-To: { "url": "https://unic.report-uri.com/r/d/csp/reportOnly”, "group": “cspro”, "max-age": 10886400 }, { "url": "https://unic.report-uri.com/r/d/csp/enforce”, "group": “csp","max-age": 10886400 }
```

Comment:
The max-age is very high. Do we have enough experience with this service, that we want to bind our website for more than half a year to this?

Reply:
No issue as the reporting end point can be removed at any time by setting the value to 0:
 3.1.4. The max-age member
The REQUIRED max-age member defines the reporting endpoint’s lifetime, as a non-negative integer number of seconds. The member’s value MUST be a number; any other type will result in a parse error.

A value of "0" will cause the endpoint to be removed from the user agent’s reporting cache.

https://www.w3.org/TR/reporting-1/

Reply:
Being able to actively flash the reporting cache is nice. Didn't know that. All good then.